changeset-builder/logs/ts-core.csv

```diff
@@ -1,4 +1,5 @@
-439e942, 2023-12-24, Enforce service name param
+ca03e82, 2023-12-24, Raise exception on unauthorised polling errors
```

Comment:
You don't have to generate these manually btw. The workflow on `main` does it.

Reply:
Oh the workflow runs on PRs! 🤦🏽 

Will fix that.

Reply:
Ahh. I have removed the change log commit and rebased `main` so it shouldn't generate for this branch again.